content/04.results.md

```diff
@@ -9,5 +9,36 @@ place the grid of aurocs here for discriminator model
 ## Random Noise Generative Model
 place the results of random label function experiment
 
-## Reconstructing Hetionet
-place figure of number of new edges that can be added to hetionet as well as edges we can reconstruct using this method
+
+### Reconstructing Hetionet
+Once the discriminiative model has been calibrated, we compared our model to a baseline model [@tag:cocoscore] (Figure {@fig:cocoscore_comparison}) and  estimated the number of edges that can be added/recalled from Hetionet v1 (Figure Figure {@fig:hetionet_reconstruction}).
```

Comment:
Does it matter that you use the calibrated model here? I don't think you're using the probabilities if I understand correctly.

Also, since the calibration may or may not have worked (kinda unclear from those plots if it's really better calibrated - why use the calibrated edges here? I could understand why you would if it worked - because then you could directly used the probabilities emitted from the model. I'm just not sure why it's used here if the calibration didn't really work.